### ISSUE TITLE
Keyboard shortcut for displaying the list of keyboard shortcuts does not work on many layouts

### ISSUE THREAD

**Issue by JustAnotherArchivist** [grafana/grafana#30024]
According to [the documentation](https://grafana.com/docs/grafana/latest/dashboards/shortcuts/):

> Grafana has a number of keyboard shortcuts available. Press Shift + ? on your keyboard to display all keyboard shortcuts available in your version of Grafana.

**Steps to reproduce**:
1. Set the device to the Swiss keyboard layout: `setxkbmap ch`
2. Try to open the keyboard shortcut list using Shift + ?.

**Actual Result**:
Nothing happens. Opening the shortcut list requires mouse navigation.

**Expected Result**:
Opens the shortcut list.

**Miscellaneous**:
I have confirmed that the same problem exists with the following layouts: `de` (German), `es` (Spanish), `fr` (French), `it` (Italian), `se` (Swedish), `sk` (Slovak), `tr` (Turkish), and `ua` (Ukranian).

I have also confirmed that the shortcut works correctly with the `gb` (British) and `us` (US) layouts.

Sidenote: technically, the shortcut is actually Shift + / on both `gb` and `us`, not Shift + ?. I tried that as well in `ch` and `de`, but since typing a slash already requires Shift on both and opens Firefox's quick search, this did not work either.

**Relevant WCAG Criteria:** [2.1.1 WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version: 7.0.0 and current play.grafana.org (7.3.6, ea06633c34)
- Data source type & version: not relevant
- User OS & Browser: Firefox 84 on Debian sid with X11
- Others: Swiss keyboard layout

**Comment by JustAnotherArchivist** [comment]
No, this isn't properly fixed, and closing issues solely because they're old is not a good strategy.

The relevant section now refers to just `?` without `Shift`. This still doesn't work on the Swiss layout (on play.grafana.org as of right now, whatever version that is running, as it doesn't seem to say that anywhere anymore or hides it very effectively). I didn't test the other layouts again, but I'd expect them to behave the same. Fortunately, there is the alternative `h` shortcut, which does work. But both should be functional.

**Comment by StefanSander3** [comment]
@tskarhed to get a question mark on german keyboard you need to press Shift + key "?ß\". And with this I wasn't able to get the keyboard shortcuts overview on https://play.grafana.org/.

Was functioning with english (US) keyboard completely fine.

Using Ubuntu 24.04.02

**Comment by L-M-K-B** [comment]
Hi @StefanSander3 and @tskarhed, I just checked the mentioned shortcut `Shift + key "?ß"` on my German keyboard and it's opening the shortcuts modal. 

Using macOS 15.3.2 and  latest versions of Brave as well as Chrome

**Comment by StefanSander3** [comment]
I was using Firefox in my last post.

With Ubuntu it was not functioning with Chrome and german keyboard layout aswell.

But: In chrome, even with german keyboard layout activated, it was working with CTRL and "'#" (which is the location of question mark on an english keyboard, not in a german one)..
